xmake/rules/qt/load.lua

```diff
@@ -70,7 +70,14 @@ function _find_static_links_3rd(target, linkdirs, qt_sdkver, libpattern)
         for _, libpath in ipairs(os.files(path.join(linkdir, libpattern))) do
             local basename = path.basename(libpath)
             -- we need ignore qt framework libraries, e.g. libQt5xxx.a, Qt5Core.lib ..
-            if not basename:startswith("libQt" .. qt_sdkver:major()) and not basename:startswith("Qt" .. qt_sdkver:major()) then
+            -- but bundled library names like libQt5Bundledxxx.a on wasm.
+            local is_bundled_library = 
```

Comment:
This name `is_bundled_library ` is ambiguous, and some of the libpcre libpng libraries built into the QT sdk may also be added.